docs/dev/api/index.rst

```diff
@@ -181,10 +181,7 @@ discovering information about Products::
    :nosignatures:
    :toctree: generate/
 
-   from_doc
```

Comment:
Why remove this? I guess it doesn't query, but still, it isn't altering anything.

Reply:
yeah I'm with @RichardScottOZ on this one, categories with duplicates are confusing